### ISSUE TITLE
Adding a pitch black night mode

### ISSUE THREAD

**Issue by sallyFoster** [vladimiry/ElectronMail#510]
Hey I just stumbled into your app, and I really like it, but I was wondering if you could add an ever darker mode than the one you have. Instead of aqua blue, you make it pitch black night mode?

<img width="1786" alt="Screen Shot 2022-04-28 at 4 51 40 PM" src="https://user-images.githubusercontent.com/104597302/165843940-6e0f7df9-87d2-4327-921c-512d9166ac22.png">

**Comment by sallyFoster** [comment]
@vladimiry Did you see this feature request?

**Comment by vladimiry** [comment]
Yes, I've seen the request. Enhancing the dark mode feature is not on my immediate TODO list.

**Comment by macbugs** [comment]
I dont see an "aqua blue" theme.  The currently supported dark themes are Carbon & Monokai.  Both appear darker on my platform than the theme used in your screenshot.

![themes2](https://user-images.githubusercontent.com/101867641/169526941-34f202b9-1caf-437a-987f-80aa1ea5278d.png)

![dark theme2](https://user-images.githubusercontent.com/101867641/169527427-5bee294f-a4e4-4a9d-9d33-9faefaec6e85.png)

There may be some inconsistency in how themes are rendered on different platforms.  Please report your OS version and ElectronMail version.  Also, if your theme palette is different than the one displayed here, give us a screenshot of that.

**Comment by vladimiry** [comment]
> the theme used in your screenshot.

The original screen in the app's dark theme (specifically displaying the [local store](https://github.com/vladimiry/ElectronMail/wiki/FAQ) view mode), not the proton's one.

**Comment by macbugs** [comment]
ok, I see the color change in that mode.  But I also see another issue here: when I activate the local storage feature, I can see in my traffic monitor that it does download something, and the size of `database.bin` has increased by several megabytes -- but when the download completes and the activity/sync indicator stops blinking, if I re-launch the app without an internet connection, it says _"loading local store"_, but defaults to online mode.  When I switch to offline mode, no mail is displayed until the connection is restored.

![offline2](https://user-images.githubusercontent.com/101867641/169539770-02843f91-4a3a-4e9e-a597-c0ef5bf29a8b.png)
![reconnect](https://user-images.githubusercontent.com/101867641/169540804-71e315ba-ca21-49a0-b47f-3770b4599042.png)

**Comment by vladimiry** [comment]
> it says "loading local store", but defaults to online mode.

The "loading local store" progress bar indicates the process of decrypting the `database.bin` and loading it to the memory in full.

> When I switch to offline mode, no mail is displayed until the connection is restored.

This is a good finding. The local store feature was originally designed as something that enables access to the email messages in the offline mode, and it worked this way before. So this is a bug.

**Comment by vladimiry** [comment]
> So this is a bug.

Should be resolved by https://github.com/vladimiry/ElectronMail/commit/05fe5684993f7aa8e1bb3fdb2291dc47c8b48019 (`wip` branch).

**Comment by sallyFoster** [comment]
Is this talk related to adding a black mode on the local store view mode?

**Comment by vladimiry** [comment]
No. That was about fixing local store view mode availability in offline mode.
